static/js/components/CommentList.jsx

```diff
@@ -306,6 +309,10 @@ const CommentList = ({
 
   comments = comments || [];
 
+  if (!includeBots) {
```

Comment:
Awesome! Thanks for adding that it's super nice.

Reply:
great.